src/components/Button/LinkButton.tsx

```diff
@@ -0,0 +1,47 @@
+import classNames from "classnames";
+import PropTypes from "prop-types";
+import React from "react";
+
+import { ComponentRegister } from "../../ComponentRegister";
+import { BaseButtonProps } from "./BaseButton";
+
+import "lbh-frontend/lbh/components/lbh-button/_button.scss";
+
+/**
+ * The proptypes for {@link LinkButton}
+ */
+export interface LinkButtonProps extends BaseButtonProps {
+  href: string;
+  children: React.ReactNode;
+}
+
+/**
+ * The link button, styled as a button, that can be used for routing
+ * within the host application.
+ */
```

Comment:
Since this feels really weird to look at in practice (a button with an `href`), it could do with some more detailed documentation about what it's for, and how it's different from the other button implementations. They could all do with a bit of cross-referencing to point out the existence of each other, and maybe some examples.